src/search-manager/SearchManager.ts

```diff
@@ -163,6 +171,7 @@ export const SearchContextProvider: React.FC<{
     searchKeywords,
```

Comment:
Some UI questions, feel free to bounce these off the UX folks if you agree :)

1. Currently, sorting by "Recently Published" also [filters out any "Never published" items](https://github.com/open-craft/frontend-app-authoring/blob/dvz/filter-by-published/src/search-manager/SearchManager.ts#L143-L145), but this is entirely hidden from the user.
   Should we make this explicit force-selecting the "Published" and "Modified since publish" filters in the UI?
2. Should the Publish Status drop-down label reveal when some filters are selected, like the Type and Tags do? 
 ![image](https://github.com/user-attachments/assets/d0dc1b2d-a68e-4e41-adc8-0bb00b59de2c)
   To make the drop-down more concise, I'd use "Status: Published" or whatever instead of "Published status: Published".

Reply:
@jmakowski1123 @lizc577 @sdaitzman @marcotuts any comments on this? ^